### PR TITLE
Update oauth-2-0-applications.rst

### PR DESCRIPTION
**Pull request by minecraftchest1**
Add OAuth endpoint URI's

#### Summary

Add information needed for people wanting to use Mattermost as an OAuth provider.

### CODE REVIEW

**Comment by minecraftchest1** [comment]
I accept the license but do not want to put my mailing address in the form.

**Comment by justinegeffen** [comment]
Thanks for the PR, @minecraftchest1, it's very helpful. :) I've added @jasonblais to this PR for feedback around the CLA. :)

File: source/developer/oauth-2-0-applications.rst [hunk 1]
```diff
@@ -85,6 +85,14 @@ Users can view a list of authorized apps from **Account Settings > Security > OA
 
 .. image:: ../images/oauth2_deauthorize_app.png
 
+OAuth login endpoints
+---------------------------------------------------------
```
Comment (justinegeffen) [inline source/developer/oauth-2-0-applications.rst:89]:
```suggestion
-------------------------------
```

File: source/developer/oauth-2-0-applications.rst [hunk 2]
```diff
@@ -85,6 +85,14 @@ Users can view a list of authorized apps from **Account Settings > Security > OA
...
+authorize uri ``/oauth/authorize``
```
Comment (justinegeffen) [inline source/developer/oauth-2-0-applications.rst:90]:
```suggestion
- Authorize URI ``/oauth/authorize``
```

File: source/developer/oauth-2-0-applications.rst [hunk 3]
```diff
@@ -85,6 +85,14 @@ Users can view a list of authorized apps from **Account Settings > Security > OA
...
+
+token uri ``/oauth/access_token``
```
Comment (justinegeffen) [inline source/developer/oauth-2-0-applications.rst:92]:
```suggestion
- Token URI ``/oauth/access_token``
```

File: source/developer/oauth-2-0-applications.rst [hunk 4]
```diff
@@ -85,6 +85,14 @@ Users can view a list of authorized apps from **Account Settings > Security > OA
...
+
+user info uri ``/api/v4/users/me``
```
Comment (justinegeffen) [inline source/developer/oauth-2-0-applications.rst:94]:
```suggestion
- User info URI ``/api/v4/users/me``
```

**Review by justinegeffen — COMMENTED** [review]
I've suggested some formatting changes, but otherwise this is good from my side. Thank you!

**Comment by minecraftchest1** [comment]
Those changes look good. I haven't ever worked with .rst files so I was not sure of the formatting.

**Comment by minecraftchest1** [comment]
Do I need to creae a changelog for this PR?

**Comment by justinegeffen** [comment]
> Do I need to creae a changelog for this PR?

No, you don't. :) The change history is in the repo for reference. 

With regard to the CLA, in order to merge this PR it does need to be signed and the address field is a mandatory one. @emilyacook, do you have any suggestions in this case? :)

This change looks good to me, so thank you for the PR!

**Review by justinegeffen — APPROVED** [review]

**Comment by minecraftchest1** [comment]
Bump.

**Review by jasonblais — APPROVED** [review]

**Comment by jasonblais** [comment]
@minecraftchest1 are you open to signing the CLA linked to at https://github.com/mattermost/docs/pull/4308#issuecomment-778547194?

I saw your note about not wanting to put your mailing address in the form - that's okay, feel free to not put anything in that field.

**Comment by minecraftchest1** [comment]
It was a requirrd field so i just put a period in the address field.
/check-cla

**Comment by minecraftchest1** [comment]
Thanks everyone.

**Comment by justinegeffen** [comment]
Thanks, @minecraftchest1! This PR hasn't yet been merged but I see it has been closed. Is it possible to reopen it on your side so we can merge the changes? :)

**Comment by minecraftchest1** [comment]
I misunderstood one of the merge messages then.

**Comment by jasonblais** [comment]
Merged, thanks @minecraftchest1 for the contribution!

**Pull request merged**